Dockerfile

```
@@ -27,6 +27,6 @@ RUN bundle install --jobs 4 --retry 3
 VOLUME "${GEMSTASH_HOME}/data"
 
 EXPOSE 9292
-COPY entrypoint.sh /
-ENTRYPOINT ["/entrypoint.sh"]
-CMD ["bundle", "exec", "gemstash", "start", "--no-daemonize"]
+USER ${GEMSTASH_USER}:${GEMSTASH_USER}
+ENTRYPOINT ["tini", "--"]
+CMD ["bundle", "exec", "gemstash", "start", "--no-daemonize", "--config-file=config.yml.erb"]
```

I think we can simplify this. A lot of what we have here is still specific to our runtime parameters which we can specify in the kubernetes manifest. That is:

1. We don't have to lock the user in the `Dockerfile` - we can force this with `securityContext`.
2. We don't have to force usage of `tini` - we've typically moved this to the manifest.
3. We don't have to specify the config - consider if someone wants to use a different file in another location.

So we can reduce it to this (especially since this is a public image):

```suggestion
CMD ["bundle", "exec", "gemstash", "start", "--no-daemonize"]
```

---

To put it another way, we can keep the image as generic as possible and focused on packaging of the app in a way anyone can use the image for their needs.